release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3478,4 +3478,32 @@ $ oc adm release info 4.13.22 --pullspecs
 [id="ocp-4-13-22-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-23"]
+=== RHSA-2023:7323 - {product-title} 4.13.23 bug fix and security update
+
+Issued: 2023-11-21
+
+{product-title} release 4.13.23, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7323[RHSA-2023:7323] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7325[RHSA-2023:7325] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.23 --pullspecs
+----
+
+[id="ocp-4-13-23-bug-fixes"]
+==== Bug fixes
+
+* Previously, Ironic was not able to provision Cisco UCS harware as a new baremetalhost because not all Redfish Virtual Media devices can be used to provision the hardware. With this release, Ironic looks at all possible devices to provision the hardware so Cisco UCS Hardware can now be provisioned using Redfish Virtual Media. (link:https://issues.redhat.com/browse/OCPBUGS-19078[*OCPBUGS-19078*])
+
+* Previously, metallb's controller restarted while having an IP assigned and unassigned LB services. Which started with an empty internal state and can move an already assigned IP to a different LB service and break workloads. With this release, metallb's controller processes the services which already have an IP assigned first. (link:https://issues.redhat.com/browse/OCPBUGS-23160[*OCPBUGS-23160*])
```

Comment:
I think the second sentence in this paragragh (regarding metalb's controller) needs to be re-written; not sure what is being said here? Also, should "services" at the end of the first sentence be singular?

Reply:
@mletalie How does it look now? Also I added a features section if you could give that a once over. Thank you!

Reply:
> @mletalie How does it look now? Also I added a features section if you could give that a once over. Thank you!

Will do! Checking now.

Reply:
LGTM, @cbippley! Thanks!